client/a8c-for-agencies/sections/sites/sites-dashboard/index.tsx

```diff
@@ -32,14 +32,21 @@ import { checkIfJetpackSiteGotDisconnected } from 'calypso/state/jetpack-agency-
 import useProductsQuery from 'calypso/state/partner-portal/licenses/hooks/use-products-query';
 import { getIsPartnerOAuthTokenLoaded } from 'calypso/state/partner-portal/partner/selectors';
 import { setSelectedSiteId } from 'calypso/state/ui/actions';
-import {
-	A4A_SITES_DASHBOARD_DEFAULT_CATEGORY,
-	A4A_SITES_DASHBOARD_DEFAULT_FEATURE,
-} from '../constants';
+import { A4A_SITES_DASHBOARD_DEFAULT_CATEGORY } from '../constants';
 import SitesDashboardContext from '../sites-dashboard-context';
 
 import './style.scss';
 
+const filtersMap: AgencyDashboardFilterMap[] = [
```

Comment:
Since we define this const in two places now (also in `sites-dashboard-v2.tsx` ) would it be worth moving it into it's own file and importing it? The only difference is that `sites-dashboard-v2.tsx` defines the const using `useMemo`, but copying that over it still works here.

Reply:
> Since we define this const in two places now (also in sites-dashboard-v2.tsx ) 

Let's consider `sites-dashboard-v2.tsx` deprecated. We won't use it anymore, and it'll be removed. 

I cherry-picked that code from this PR: https://github.com/Automattic/wp-calypso/pull/88073 (probably, this PR will be closed after we pick some of the code there)